tests/trainer/flags/test_min_max_epochs.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from pytorch_lightning import Trainer
+from tests.base.boring_model import BoringModel
+
+
+# @pytest.mark.parametrize("min_epochs", [None, 2])
+# @pytest.mark.parametrize("max_epochs", [None, 3])
+# @pytest.mark.parametrize("min_steps", [None, 20])
+# @pytest.mark.parametrize("max_steps", [None, 100])
+@pytest.mark.parametrize(
+    ["min_epochs", "max_epochs", "min_steps", "max_steps"],
+    [
+        pytest.param(None, 5, None, None),
+        pytest.param(None, None, None, 100),
+        pytest.param(None, 5, None, 100),
+        pytest.param(None, None, 10, 100),
+        pytest.param(1, 5, None, None),
+        pytest.param(1, None, None, 100),
+        pytest.param(None, 5, 10, None),
```

Comment:
could we lower the numbers just a little bit to speed up the test?

Reply:
Use plain tuples instead of `pytest.param` if its advantages aren't necessary